checks/evaluation/branch_protection.go

```diff
@@ -371,18 +341,24 @@ func adminThoroughReviewProtection(branch *clients.BranchRef, dl checker.DetailL
 	// Only log information if the branch is protected.
 	log := branch.Protected != nil && *branch.Protected
 
-	if branch.BranchProtectionRule.RequiredPullRequestReviews.DismissStaleReviews != nil {
-		// Note: we don't inrecase max possible score for non-admin viewers.
+	if branch.BranchProtectionRule.RequiredPullRequestReviews == nil {
+		// If the repo doesn't require PRs, we still have to discount score for not dismissing stale reviews.
 		max++
```

Comment:
You're correct. My reasoning to write the current code was the following case:
- Scorecard is run with admin token on a repo that don't require PRs. The repo should still be penalized for not dismissing stale reviews (i.e., we should have the max++)

And I think that could indeed be a problem **if we don't have tiers**, but currently it's not a problem because if someone doesn't require PRs (tier 2) we won't look if they dismiss stale reviews(tier 5).

But the problem you pointed out would affect the current scoring and I didn't realize it >< . I'll revert it.